working/2364 - primary constructors/feature-specification.md

```diff
@@ -361,14 +397,28 @@ constants *(holding parameter default values)*.
 Where no processing is mentioned below, _D2_ is identical to _D_. Changes
 occur as follows:
 
+Assume that `p` is an optional formal parameter in _D_ that does not have a
+declared type, but it does have a default value whose static type in the
+empty context is a type `T` which is not `Null`. In that case `p` is
```

Comment:
(Probably a good call to special-case `Null`. Although writing `var x = null` is unnecessary, you can just write `var x`, it's also good that the two actually mean the same thing.)

Reply:
Ack!